### PR TITLE
added initial h2o spec file

### PR DESCRIPTION
**Pull request by c1mckay**
First try in pushing the spec for h2Oai.  It's pretty simple to setup, and I have the Dockerfile as well.

### CODE REVIEW

File: h2oai/h2oai.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "id": "h2oai",
+    "key": "h2oai",
+    "label": "H2O AI",
+    "description": "AI for the Digital Brain",
+    "info": "https://www.h2o.ai/",
```
Comment (craig-willis) [inline h2oai/h2oai.json:6]: "info" needs to be a URL to a page in the NDS Confluence:
https://nationaldataservice.atlassian.net/wiki/display/NDSC/NDS+Labs+Services

Create a new service entry as a subpage of "NDS Labs Services". Copy the "Template" example and add some very minimal descriptive information (you don't need to provide everything in the template, but certainly how to run the examples as outlined in the JIRA ticket would be nice).

File: h2oai/h2oai.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "logo": "https://h2o2016.wpengine.com/wp-content/themes/h2o2016/images/H2O_logo_yellow.svg",
+    "image": {
+        "registry" : "",
+        "name" : "h2oai",
```
Comment (craig-willis) [inline h2oai/h2oai.json]: This needs to be ndslabs/h2o (same as the docker hub image I created)

File: h2oai/h2oai.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        }
+    ],
+    "repositories":  [
+        {
+            "url": "https://raw.githubusercontent.com/h2oai/h2o-3/master/Dockerfile",
+            "type": "git"
+        }
+    ],
+    "developerEnvironment" : "",
+    "volumeMounts":[],
```
Comment (craig-willis) [inline h2oai/h2oai.json]: I'm surprised that there are no volume mounts (i.e., no data stored by the tool in the container).  As far as I can tell, data/notebooks must be uploaded/downloaded to local computer.  Might be worth confirming.

File: h2oai/h2oai.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        "initialDelay": 30,
+        "timeout" : 600
+    },
+    "resourceLimits": {
+        "cpuMax": 1000,
+        "cpuDefault": 100,
+        "memMax": 1024,
+        "memDefault": 512
+    },
+    "tags": []
```
Comment (craig-willis) [inline h2oai/h2oai.json]: Consider adding tags from vocab/tags.json. You might consider adding a tag for AI or just applying some of the existing tags, if relevant.

File: h2oai/h2oai.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+}
```
Comment (craig-willis) [inline h2oai/h2oai.json]: Add  key/value   
   "authRequired": true
We use this to force authentication for any application that doesn't have it's own notion of auth.

File: h2oai/h2oai.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        {
+            "url": "https://raw.githubusercontent.com/h2oai/h2o-3/master/Dockerfile",
+            "type": "git"
+        }
+    ],
+    "developerEnvironment" : "",
+    "volumeMounts":[],
+    "readinessProbe" : {
+        "type" : "http",
+        "path" : "",
```
Comment (craig-willis) [inline h2oai/h2oai.json]: Need a path here -- "/" should work.

File: h2oai/h2oai.json (hunk 3 above)
Comment (c1mckay) [inline h2oai/h2oai.json]: @craig-willis how does this work for other services that use volumeMount?

**Pull request closed without merging**